src/re_bytes.rs

```diff
@@ -986,7 +987,7 @@ pub trait Replacer {
     /// be beneficial to avoid finding sub-captures.
     ///
     /// In general, this is called once for every call to `replacen`.
-    fn no_expansion<'r>(&'r mut self) -> Option<Cow<'r, [u8]>> {
+    fn no_expansion(&mut self) -> Option<Cow<[u8]>> {
```

Comment:
Please put the lifetimes back. I don't mind elision in simple cases, but I'd rather the lifetime parameter be explicit here, *especially* in the trait definition.